M2/Macaulay2/d/parse.d

```diff
@@ -210,7 +210,6 @@ export augmentedAssignmentCode := {+
     oper:Symbol,
     lhs:Code,
     rhs:Code,
-    info:Symbol, -- variable name or operator
```

Comment:
Is there any reason to keep this for debugging purposes?

Reply:
This was a hack that I came up when adding augmented assignment.  There were two unrelated cases where we needed some information about a symbol that wasn't being stored in the Code object:

* The variable name for `globalMemoryReferenceCode` and `threadMemoryReferenceCode`, which currently just store the frame index of the variable.
* The binary/unary operator for `binaryCode` and `unaryCode`, which currently just store the function associated with the operator.

With this PR, the proposal is that these various Code objects now store the symbols themselves.  So the symbol that was in `info` is still there, but now inside `lhs`.